pkg/controller/endpoint/config/v1alpha1/defaults.go

```diff
@@ -33,4 +36,9 @@ func RecommendedDefaultEndpointControllerConfiguration(obj *kubectrlmgrconfigv1a
 	if obj.ConcurrentEndpointSyncs == 0 {
 		obj.ConcurrentEndpointSyncs = 5
 	}
+
+	zero := metav1.Duration{}
+	if obj.EndpointUpdatesBatchPeriod == zero {
+		obj.EndpointUpdatesBatchPeriod = metav1.Duration{Duration: 0 * time.Second}
```

Comment:
Not sure I understand this - isn't it no-op?

Reply:
You are right -- removed.